### PR TITLE
Add fallback for missing ReferringPhysicianName, add qualitative_evaluations to test_sr

### PR DESCRIPTION
**Pull request by fedorov**

### CODE REVIEW

**Comment by fedorov** [comment]
On a related note, it took me quite some time to figure out that I need to initialize `RelationshipType` manually (as in https://github.com/MGHComputationalPathology/highdicom/pull/66/files#diff-346e4cd4f40101300bc7800229e1371f1428b2d395ec4207d3ec788e7d4f98e3R1875) when passing qualitative evaluations (as in this example here: https://github.com/MGHComputationalPathology/highdicom/pull/66/files#diff-346e4cd4f40101300bc7800229e1371f1428b2d395ec4207d3ec788e7d4f98e3R1894).

In this particular case, due to the template constraints, the relationship of that `CODE` content item will always be `CONTAINS`. It would make a lot more sense if this relationship was automatically initialized somewhere here: https://github.com/MGHComputationalPathology/highdicom/blob/master/src/highdicom/sr/templates.py#L1360-L1366. Hopefully this addition to the test will make it easier for the next person trying to do what I was doing!

File: src/highdicom/legacy/sop.py [hunk 1]
```diff
@@ -457,7 +457,8 @@ def __init__(
             study_id=ref_ds.StudyID,
             study_date=ref_ds.StudyDate,
             study_time=ref_ds.StudyTime,
-            referring_physician_name=ref_ds.ReferringPhysicianName,
+            referring_physician_name=getattr(
+                ref_ds, "ReferringPhysicianName", None),
```
Comment (hackermd) [inline src/highdicom/legacy/sop.py:461]:
```suggestion
            referring_physician_name=getattr(
                ref_ds, 'ReferringPhysicianName', None
            ),
```

File: src/highdicom/seg/sop.py [hunk 2]
```diff
@@ -255,7 +255,8 @@ def __init__(
             study_id=src_img.StudyID,
             study_date=src_img.StudyDate,
             study_time=src_img.StudyTime,
-            referring_physician_name=src_img.ReferringPhysicianName,
+            referring_physician_name=getattr(
+                src_img, "ReferringPhysicianName", None),
```
Comment (hackermd) [inline src/highdicom/seg/sop.py:259]:
```suggestion
            referring_physician_name=getattr(
                src_img, 'ReferringPhysicianName', None
            ),
```

File: src/highdicom/sr/sop.py [hunk 3]
```diff
@@ -134,7 +134,8 @@ def __init__(
             study_id=evidence[0].StudyID,
             study_date=evidence[0].StudyDate,
             study_time=evidence[0].StudyTime,
-            referring_physician_name=evidence[0].ReferringPhysicianName,
+            referring_physician_name=getattr(
+                evidence[0], "ReferringPhysicianName", None),
```
Comment (hackermd) [inline src/highdicom/sr/sop.py:138]:
```suggestion
            referring_physician_name=getattr(
                evidence[0], 'ReferringPhysicianName', None
            ),
```

**Review by hackermd — CHANGES_REQUESTED** [review]
Thanks @fedorov. Good catch! I have only a few minor comments regarding coding style (single quotes instead of double quotes and closing brackets on new line).

**Comment by hackermd** [comment]
Regarding relationship type, see related discussion https://github.com/MGHComputationalPathology/highdicom/issues/61

**Comment by fedorov** [comment]
Thanks Markus - your suggestions applied.

Indeed, I actually even commented in #61, but did not dig into the main topic of the issue. I will follow up in that thread. Thanks for making the reference!

**Review by hackermd — APPROVED** [review]

**Pull request merged**